app/src/main/java/com/tari/android/wallet/ui/fragment/onboarding/CreateWalletFragment.kt

```diff
@@ -89,19 +88,30 @@ class CreateWalletActivity : BaseActivity() {
     @JvmField
     var whiteColor = 0
 
-    override val contentViewId = R.layout.activity_create_wallet
-
     private val uiHandler = Handler()
 
-    override fun onCreate(savedInstanceState: Bundle?) {
-        super.onCreate(savedInstanceState)
-        overridePendingTransition(android.R.anim.fade_in, android.R.anim.fade_out)
+    override val contentViewId = R.layout.fragment_create_wallet
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
         init()
     }
 
+    private fun init() {
+        rootView.post { startViewAnimation() }
+        UiUtil.setProgressBarColor(progressBar, whiteColor)
+        setVideoDimension()
+    }
+
+    private fun showCreateEmojiIdView() {
```

Comment:
Not sure about the naming, it sounds like it's showing a view inside itself, whereas it's navigating to another fragment.